src/System.Data.Common/tests/System/Data/DataTableTest.cs

```diff
@@ -3776,12 +3776,12 @@ public void WriteXmlSchema_ForignKeyConstraint()
             StringWriter sw1 = new StringWriter();
             ds1.Tables[0].WriteXmlSchema(sw1);
             string xml1 = sw1.ToString();
-            Assert.True(xml1.IndexOf(@"<xs:unique name=""Constraint1"">") != -1);
+            Assert.True(xml1.Contains(@"<xs:unique name=""Constraint1"">"));
```

Comment:
what about `Assert.Contains (@"<xs:unique name=""Constraint1"">", xml1));`

Reply:
Actually, that uses current culture. You either have to pass the StringComparison explicitly, or use `AssertExtensions.Contains` which uses Ordinal.

Reply:
Cool, didn't know about those methods.